website/docs/d/google_container_registry_repository.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: "google"
+page_title: "Google: google_container_registry_repository"
+sidebar_current: "docs-google-datasource-container-repo"
+description: |-
+  Get URLs for a given project's container registry repository.
+---
+
+# google\_container\_registry\_repository
+
+This data source fetches the project name, and provides the appropriate URLs to use for container registry for this project.
+
+The URLs are computed entirely offline - as long as the project exists, they will be valid, but this data source does not contact GCR at any point.
```

Comment:
If you want to use the GCR acronym, you should spell it out fully the first time you use it and add the acronym between parentheses after it.